pagerduty/pagerduty.go

```diff
@@ -77,7 +77,7 @@ func NewClient(config *Config) (*Client, error) {
 		config.BaseURL = defaultBaseURL
 	}
 
-	config.UserAgent = "heimweh/go-pagerduty(terraform)"
+	config.UserAgent = "heimweh/go-pagerduty(terraform-caching)"
```

Comment:
The client library is still the library for both the regular provider traffic and the cached traffic. I'd rather this remain `(terraform)` unless you wanted to put logic in here that passed a different `UserAgent` header based on if the environment variables were set.

Reply:
@stmcallister you are right. I believe it should remain `(terraform)` as well. Pushed.